cmd/server/pactasrv/conv/pacta_to_oapi.go

```diff
@@ -236,3 +236,73 @@ func PortfolioGroupToOAPI(pg *pacta.PortfolioGroup) (*api.PortfolioGroup, error)
 func PortfolioGroupsToOAPI(pgs []*pacta.PortfolioGroup) ([]*api.PortfolioGroup, error) {
 	return convAll(pgs, PortfolioGroupToOAPI)
 }
+
+func auditLogActorTypeToOAPI(i pacta.AuditLogActorType) (api.AuditLogActorType, error) {
+	return api.AuditLogActorType(string(i)), nil
```

Comment:
Does the generated `api` code not produce unique enums? Seems like it'd be better to explicitly map them, instead of implicitly expecting them to stay in sync. Same below

Reply:
SGTM, done.

Reply:
Great call, found several small typos this way.